website/cue/reference/components/sinks.cue

```diff
@@ -236,7 +236,7 @@ components: sinks: [Name=string]: {
 													type: string: {
 														examples: [
 															"""
-															[{ "type": "record", "name": "log", "fields": [{ "name": "message", "type": "string" }] }]
+															{ "type": "record", "name": "log", "fields": [{ "name": "message", "type": "string" }] }
```

Comment:
I'm curious what necessitated this change.

Reply:
Let me explain. The previous change is made here: https://github.com/vectordotdev/vector/pull/15145

As you see, the Avro example was wrong. But Jesse was merged the PR too quickly and I didn't remove outer brackets in the PR.

This field describes the Avro scheme. You could check e.g. here https://www.tutorialspoint.com/avro/avro_schemas.htm that the scheme doesn't need to be included into the `[]` brackets. That's why I've removed them.

Of course I could do it in one more separate PR but... you know, just a little fix in the docs :)